editor/renames_map_3_to_4.cpp

```diff
@@ -579,7 +579,7 @@ const char *RenamesMap3To4::gdscript_function_renames[][2] = {
 	// Remember to add them to the builtin_types_excluded_functions variable, because for now these functions cannot be listed.
 	//	{ "empty", "is_empty" }, // Array -- Used as custom rule. Be careful, this will be used everywhere.
 	//	{ "remove", "remove_at" }, // Array -- Breaks Directory and several more.
-	{ "clamped", "clamp" }, // Vector2 -- Be careful, this will be used everywhere.
+	{ "clamped", "limit_length" }, // Vector2 -- Doc already said for 3.5 clamped is deprecated and limit_lenght should be used
```

Comment:
The comment (and the commit message) has a typo, but I'd simply remove it, it's the correct rename and doesn't require justification in the code.

Reply:
thank you for the input, I reverted the comment to what it was previously. I am 100% sure the rename is correct after this change, but would expect to get confirmation from anyone else